### PR TITLE
Support rawResultMode=True option

### PR DESCRIPTION
**Pull request by kriii**
Support `rawResultMode=True` option.

* Related issue
ainblockchain/ain-blockchain#1053

### CODE REVIEW

File: tests/test_ain.py [hunk 1]
```diff
@@ -152,12 +149,15 @@ async def testGetBalance(self):
         ain.wallet.addAndSetDefaultAccount(accountSk)
         self.assertGreaterEqual(await ain.wallet.getBalance(), 0)
 
+    # TODO(kriii): This test causes failure for now,
```
Comment (platfowner) [inline tests/test_ain.py:152]: question) can this be fixed by adding some env param setting to start_local_blockchain.sh?

Comment (kriii) [reply tests/test_ain.py:152]: Yes, increasing `TX_POOL_SIZE_LIMIT_PER_ACCOUNT` or `FREE_TX_POOL_SIZE_LIMIT_RATIO_PER_ACCOUNT` works.

Comment (platfowner) [reply tests/test_ain.py:152]: Then could you include that change in this PR.
fyi, `ENABLE_EXPRESS_RATE_LIMIT=false` was added for similar reasons.

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by kriii** [comment]
Now merged! Thank for the review.

**Pull request merged**